Sources/TokamakDOM/Views/ScrollView.swift

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 06/29/2020.
+//
+
+import TokamakCore
+
+public typealias ScrollView = TokamakCore.ScrollView
+
+extension ScrollView: ViewDeferredToRenderer, SpacerContainer {
+  var axis: SpacerContainerAxis {
+    if axes.contains(.horizontal) {
+      return .horizontal
+    } else {
+      return .vertical
+    }
+  }
+
+  public var deferredBody: AnyView {
+    let scrollX = axes.contains(.horizontal)
+    let scrollY = axes.contains(.vertical)
+    // FIXME: I can only hide the scrollbars on Firefox
+    // Safari/Chrome require a `::-webkit-scrollbar` selector, which can't be done with inline styles
```

Comment:
Could you make a global CSS rule like

```css
._tokamak-indicatorless-scrollview::-webkit-scrollbar {
  width: 0;
  height: 0;
}
._tokamak-indicatorless-scrollview::scrollbar {
  width: 0;
  height: 0;
}
```

Reply:
Yeah, that would work. We currently don't inject any styles to the DOM, but we could add one. @MaxDesiatov Thoughts?

Reply:
Yeah, `DOMRenderer` would be a good entrypoint for that 👍